trieste/utils/pareto.py

```diff
@@ -47,7 +48,7 @@ def non_dominated(observations: TensorType) -> Tuple[TensorType, TensorType]:
     return tf.boolean_mask(observations, dominance == 0), dominance
 
 
-class BoundedVolumes:
+class _BoundedVolumes:
```

Comment:
As discussed with @satrialoka, it may be suitable to make this class internal, CMIIW.

Reply:
What does this mean?

Reply:
The  `BoundedVolumes` class is more only used internally to store the index of bounds.
And since we could provide the `get_hypercell_bounds` for users to directly get the partitioned cell bounds for acquisition function building, this class may not need to expose outside `pareto.py`.

Reply:
Makes sense!